cogs/SquadQueue.py

```diff
@@ -730,10 +880,15 @@ async def write_history(mogi: Mogi, history_channel: discord.TextChannel):
         """Writes the teams, tier and average of each room per hour."""
         try:
             if mogi is not None:
-                await history_channel.send(f"{discord.utils.format_dt(mogi.start_time)} Rooms")
+                # I want the time displayed to be at the hour but the current way reads as :55 minutes on each queue which looks bad...
```

Comment:
I feel this is a result of an incorrect config setup. Does MK8DX have a posted schedule? If so, do those times denote when the queue would regularly close (assuming no extension)?

Reply:
My current config for mk8 is 

"FIRST_EVENT_TIME": 0,
"QUEUE_OPEN_TIME": 60,
"JOINING_TIME": 55,
"EXTENSION_TIME": 3,
    
and my intention is to have each queue open at :00, each queue close at :55, maybe i dont really understand what config would accomplish that so if you know please tell

Reply:
Let’s discuss on Discord.

Reply:
[https://github.com/emilp-595/QueueBot/pull/15/commits/573ad61305e91d0f430a67e56542e1a202296f62](https://github.com/emilp-595/QueueBot/pull/15/commits/573ad61305e91d0f430a67e56542e1a202296f62)